.github/workflows/lint-pr-title.yml

```diff
@@ -7,7 +7,7 @@ name: Lint PR Title
 
 on:
   pull_request:
-    types: [opened, synchronize, reopened, ready_for_review]
+    types: [opened, synchronize, reopened, ready_for_review, edited]
```

Comment:
This fixes an issue with the run not re-triggering if someone updates the PR title, which was annowing